<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommuteWidget</class>
 <widget class="QWidget" name="CommuteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="arrivalLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>390</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Estimated Arrival Time</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trafficLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>180</y>
     <width>390</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Traffic Volume</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="durationValue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>65</y>
     <width>360</width>
     <height>180</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>160</pointsize>
    </font>
   </property>
   <property name="text">
    <string>135</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="durationLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Commute Duration</string>
   </property>
  </widget>
  <widget class="QLabel" name="unitLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>360</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>minutes</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="arrivalValue">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>390</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12:00 pm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="trafficValue">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>390</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Moderate</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
